### ISSUE TITLE
ADA compliance issues while using screen reader

### ISSUE THREAD

**Issue by bhargavgunreddy** [ag-grid/ag-grid#2523]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => ag-grid is not ADA compliant as per the feedback from ADA compliance report on
out application which uses ag-grid.

**Current behavior**
 If column header is Location and cell value is Newyork, it should read as "location newyork".

**Expected behavior**
Instead it just reads out "column 2 newyork".


**Minimal reproduction of the problem with instructions**
Create a simple grid and try to navigate through it using screen reader with audio/speech viewer on.

**What is the motivation / use case for changing the behavior?**
I started using ag-grid after going through their documentation which says it is ADA compliant. 
I am using ag-grid in a client facing application and it has to be ADA compliant. 
Our application went through ADA compliance review and got the below feedback.

**Feedback:**
 Since it is not provided in the proper table structure, so screen reader 
 user wont be able to relate the content with the respective table   header.    
 Content is not provided in table structure as recommended. 
Also, now the   table content is sortable so that should be accessible to screen reader   users.     
1.Provide scope attribute to the column headers.     
2.Provide descriptive caption for the table that can be made available only   for the screen reader user. Provide aria-live="polite" attribute to  the caption.     
3.Ensure that caption should change on each sort dynamically :   "Action Items(table name) sorted by "Task" (Column name ) in   Ascending order".    
4.Also,ensure that headers are  provided as buttons, so that user can understand that it is clickable.     
5. Remove unnecessary aria-attribute like role=grid,aria-rowindex,   tab-index etc.     




**Please tell us about your environment:**
Windows 7, IntelliJ, NVDA screen reader, Chrome

* **ag-Grid version:** X.X.X
15.0.0


* **Browser:**
Google Chrome

Angular 5.2.3

**Comment by bhargavgunreddy** [comment]
Surprising to see the grid inline on page ( https://www.ag-grid.com/javascript-grid-accessibility/) does not read out column header names when i navigate through cells, but if i open it in a new tab it does read out the column header name along with cell value.
Can you please tell if there is any difference in grid options fro both ?

**Comment by RodSmithAG** [comment]
Hi,

Thank you for your email. We are aware of this issue and have a number of feature requests in our backlog like:

AG-1976  -  Grouping - Add Accessibility attributes across grouping elements

You can follow the status of feature requests, bugs and releases using our pipeline: 
https://www.ag-grid.com/ag-grid-pipeline/

If you are an ag-Grid Enterprise Customer, please raise this in our Zendesk support system - contact info@ag-grid.com for access. This is our primary channel for Support.

**Comment by bhargavgunreddy** [comment]
Can you at least read my comment above ? 
I was asking if there is any difference between the grid in line and the grid that opens in new tab. because one works fine while the other doesn't.

Anyways 
Thanks

**Comment by ludakhris** [comment]
Guess not lol absolutely TERRIBLE support.
